src/Shared/E2ETesting/SeleniumStandaloneServer.cs

```diff
@@ -111,7 +111,7 @@ private static async Task InitializeInstance(ITestOutputHelper output)
         var psi = new ProcessStartInfo
         {
             FileName = "npm",
-            Arguments = $"run selenium-standalone start -- --config \"{seleniumConfigPath}\" {chromeDriverArg} -- -port {port}",
+            Arguments = $"run selenium-standalone start -- --config \"{seleniumConfigPath}\" {chromeDriverArg} -- --host localhost --port {port}",
```

Comment:
It's important to pass the host explicitly here, otherwise it uses 127.0.0.1 and other software can interfere with it. (specifically, the local kubernetes cluster in docker desktop)